Harmonize nav and fix contribute (#4794)
owncloud/docs: modules/ROOT/pages/how_to_contribute.adoc

= How to Contribute to the ownCloud Documentation
:toc: right

:asciidoc-syntax-url: https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/
:antora-platform-url: https://docs.antora.org/
:bpg-url: https://github.com/owncloud/docs/blob/master/docs/best-practices.md
:github-docs-url: https://github.com/owncloud

:description: ownCloud welcomes all helpful contributions to the documentation, whether they’re reporting issues, adding new material, or writing tips and tricks. 

{description}

== Documentation  Prerequisites

If you wish to contribute to the official documentation manuals, you must:

* Have a https://github.com/[GitHub] account.

> * Get used to the framwork

Reviewer: ```suggestion
* Get used to the framework
```

** The ownCloud documentation is written in the {asciidoc-syntax-url}[AsciiDoc file format]
and is powered by the {antora-platform-url}[Antora documentation platform].

> ** Read the https://github.com/owncloud/docs/blob/master/docs/getting-started.md[documentation] for instructions on setting up your build environment and becoming familiar with the file format and the build system that powers the documentation.

Reviewer 2: ```suggestion
** Read the https://github.com/owncloud/docs/blob/master/docs/getting-started.md[documentation] for instructions on setting up your build environment and become familiar with the file format and the build system that powers the documentation.
```

** When contributing to the documentation, please follow our {bpg-url}[Best Practices and Tips guide].

* Fork the official repositories as described below, write your changes and create a pull request. 

== Documentation Sources

The locations for the sources can be found at:

> * The *main page* source which refrences all other documentations is found at {github-docs-url}/docs[owncloud/docs].

Reviewer: ```suggestion
* The *main page* source which references all other documentation is found at {github-docs-url}/docs[owncloud/docs].
```

* For *Infinite Scale*, you find the documentation source in {github-docs-url}/docs-ocis[owncloud/docs-ocis].

> ** If you have any suggestions regarding development, see at https://owncloud.dev[owncloud.dev] and the sources at {github-docs-url}/ocis[owncloud/ocis].

Reviewer 2: ```suggestion
** If you have any suggestions regarding development, see https://owncloud.dev[owncloud.dev] and the sources at {github-docs-url}/ocis[owncloud/ocis].
```

* For *ownCloud Server*, the `Administration` and `Developer` manuals are in {github-docs-url}/docs[owncloud/docs-server].

> * The *web UI* for both ownCloud Server and Infinite Scale is located in {github-docs-url}/docs-webui[owncloud/docs-webui].

Reviewer: ```suggestion
* The *web UI* documentation source for both ownCloud Server and Infinite Scale is located in {github-docs-url}/docs-webui[owncloud/docs-webui].
```

> * For the *Desktop Sync Client* the documentaion source is located in {github-docs-url}/docs-client-desktop[owncloud/docs-client-desktop].

Reviewer: ```suggestion
* The *Desktop Sync Client* documentation source is located in {github-docs-url}/docs-client-desktop[owncloud/docs-client-desktop].
```

> * For the *Android app* the documentaion source is located in {github-docs-url}/docs-client-android[owncloud/docs-client-android].

Reviewer: ```suggestion
* The *Android app* documentation source is located in {github-docs-url}/docs-client-android[owncloud/docs-client-android].
```

> * For the *iOS app* the documentaion source is located in {github-docs-url}/docs-client-ios-app[owncloud/docs-client-ios-app].

Reviewer: ```suggestion
* The *iOS app* documentation source is located in {github-docs-url}/docs-client-ios-app[owncloud/docs-client-ios-app].
```

> * Finally, when contributing to the *UI* used by all documentation repositories, the is source located at {github-docs-url}/docs-ui[owncloud/docs-ui].

Reviewer: ```suggestion
* Finally, when contributing to the *UI* used by all documentation repositories, the source is located at {github-docs-url}/docs-ui[owncloud/docs-ui].
```
